using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -525,23 +599,23 @@ ACPIdentity.getUrlVariables().then(urlVariables => console.log("AdobeExperenceSD
 {% endtab %}
 {% endtabs %}
 
-## Get identifiers
+## Get identifiers<a id="getIdentifiersTitle"></a>
 
 {% tabs %}
 {% tab title="Android" %}
-### getIdentifiers
-
-Returns all customer identifiers that were previously synced with the Adobe Experience Cloud.
+### getIdentifiers<a id="getIdentifiers-java"></a>
 
-The values are returned through the [AdobeCallback](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/identity/identity-api-reference#adobecallback).
+This `getIdentifiers()` API returns all customer identifiers that were previously synced with the Adobe Experience Cloud through the [AdobeCallback](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/identity/identity-api-reference#adobecallback).
 
 #### **Syntax**
 
 ```java
 public static void getIdentifiers(final AdobeCallback<List<VisitorID>> callback);
 ```
 
-#### **Example**
+o   *callback* is a method which will be invoked once the customer identifiers are available.
```

Comment:
- Reword to say:
"*callback* is invoked after the customer identifiers are available."